test/advanced.js

```diff
@@ -174,7 +174,7 @@ test('should be able to access the query to determine that users have insufficie
 })
 
 test('should support being registered multiple times', async (t) => {
```

Comment:
could we have a separate test for the `authMerge` fix rather than changing an existing test?